apps/api/scripts/main-faulty-logs-to-excel.py

```
@@ -0,0 +1,140 @@
+                                message_index = list(first_obj.keys()).index("message") + 1
+                            except ValueError:
+                                # If message key is missing, we cannot proceed with this entry
+                                print(f"Skipping line {line_num} in {log_file}: 'message' key missing in header {first_obj}")
+                                continue # Skip to next line
+
+                            # Try to find source index
+                            try:
+                                source_index = list(first_obj.keys()).index("source") + 1
+                            except ValueError:
+                                # 'source' key not found in header, source remains "Unknown Source"
+                                pass
```

Ruff (0.11.9)

Use `contextlib.suppress(ValueError)` instead of `try`-`except`-`pass`

(SIM105)